### ISSUE TITLE
Mobile Chrome detected as Opera

### ISSUE THREAD

**Issue by raapperez** [lancedikson/bowser#253]
I think it's because there is an OPR in the user agent, but I can't understand why it have this word in... It's coincidence or it has some relation with Opera?

`window.navigator.userAgent` of the browser is:
```
Mozilla/5.0 (Linux; Android 8.0.0; LG-H870 Build/OPR1.170623.032) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.91 Mobile Safari/537.36 
```
 And it's detected like Opera (without version)
 However, the real name of the browser is Chrome version 68

**Comment by lancedikson** [comment]
Hi, it seems you’re using an older version of the library. Have you tried upgrading to the latest minor release? This issue has been fixed since #193. Check it out, please, to make sure that it’s not the same issue.

**Comment by raapperez** [comment]
It looks like that is fixed in branch of the version 1.x but not in master.

**Comment by lancedikson** [comment]
It's been fixed in 2.0.0-beta.3
